cmd/server/main.go

```diff
@@ -111,17 +111,27 @@ func main() {
 		cfg.NetworkName = networkName
 	}
 
-	network := &types.NetworkIdentifier{
+	networkP := &types.NetworkIdentifier{
+		Blockchain: service.BlockchainName,
+		Network:    cfg.NetworkName,
+		SubNetworkIdentifier: &types.SubNetworkIdentifier{
+			Network: mapper.PChainNetworkIdentifier,
+		},
+	}
+	networkC := &types.NetworkIdentifier{
 		Blockchain: service.BlockchainName,
 		Network:    cfg.NetworkName,
 	}
 
 	asserter, err := asserter.NewServer(
-		mapper.OperationTypes,               // supported operation types
-		true,                                // historical balance lookup
-		[]*types.NetworkIdentifier{network}, // supported networks
-		[]string{},                          // call methods
-		false,                               // mempool coins
+		mapper.OperationTypes,      // supported operation types
+		true,                       // historical balance lookup
+		[]*types.NetworkIdentifier{ // supported networks
+			networkP,
+			networkC,
+		},
+		[]string{},                 // call methods
+		false,                      // mempool coins
```

Comment:
These comments are causing the linting to fail.

Reply:
Fixed! :)